src/TypeHandlers/TH_Tokenized.cpp

```diff
@@ -34,9 +34,15 @@ namespace tivars
     /* TODO: handle TI-Innovator Send( exception for in-strings tokenization (=> not shortest tokens) */
     data_t TH_Tokenized::makeDataFromString(const std::string& str, const options_t& options)
     {
-        (void)options;
         data_t data;
 
+        const auto& optIter = options.find("_detect_strings");
+        bool detect_strings = true;
+        if (optIter != options.end())
+        {
+            detect_strings = optIter->second;
+        }
+
```

Comment:
Nah, you can use something like:

```suggestion
        const bool detect_strings = !has_option(options, "detect_strings") || options.at("detect_strings") != 0
```

Reply:
Ah, neat. I'm not super familiar with C++, so I just copied how the generic reals did it.

Reply:
Yeah, I should also use `has_option` there :D